modules/security-hardening-how.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-hardening.adoc
+
+[id="security-hardening-how_{context}"]
+
+= Choosing how to harden RHCOS
```

Comment:
So we actually have attributes for RHCOS, so that it's written properly. And also so that it shows as RH CoreOS in enterprise, but Fedora CoreOS in origin.

So anywhere that you use RHCOS, please replace with {op-system}.

And anywhere that you write it out, please replace with {op-system-first}. This will write it out as "Red Hat Enterprise Linux CoreOS (RHCOS)".

Reply:
Cool. Okay. Done.